elasticsearch_metrics/management/commands/sync_metrics.py

```diff
@@ -1,12 +1,37 @@
-from django.core.management.base import BaseCommand
+from django.core.management.base import BaseCommand, CommandError
+
+from elasticsearch_metrics.registry import registry
 
 
 class Command(BaseCommand):
     help = "Ensures index templates exist for all Metric classes."
 
     def add_arguments(self, parser):
-        pass
+        parser.add_argument(
+            "app_label",
+            nargs="?",
```

Comment:
What is the nargs argument? I see examples of people adding that when they are creating their own django-admin commands, but not sure how we would use it.

Reply:
If we were going to pass in the app_label, it would look like this? `python manage.py sync_metrics dummyapp`.  Do we want to change so there's a label for that argument in case we ever have multiple arguments?

Reply:
> What is the nargs argument? 

Django mgmt commands use the argparse module from the standard library. Docs on nargs: https://docs.python.org/3/library/argparse.html#nargs 

> Do we want to change so there's a label for that argument in case we ever have multiple arguments?

Can you clarify? Are you suggesting that we support `python manage.py sync_metrics app1 app2...`?

Reply:
no, thinking more, that's probably not necessary

Reply:
OK. I should mention: I copied this call from Django's `migrate` command, which doesn't support passing multiple apps.